pkg/attacher/connection.go

```diff
@@ -103,16 +90,16 @@ func (c *csiConnection) SupportsControllerPublish(ctx context.Context) (supports
 	return supportsControllerPublish, supportsPublishReadOnly, nil
 }
 
-func (c *csiConnection) SupportsPluginControllerService(ctx context.Context) (bool, error) {
-	caps, err := connection.GetPluginCapabilities(ctx, c.conn)
+func (c *attacher) SupportsPluginControllerService(ctx context.Context) (bool, error) {
```

Comment:
same here, I don't think `SupportsPluginControllerService ` needs to be part of attacher, let's just move this as an unexported function in main.go?